query.go

```diff
@@ -77,11 +80,12 @@ type queryChannelResponseData struct {
 func (c *Client) QueryChannels(q *QueryOption, sort ...*SortOption) ([]*Channel, error) {
 	qp := queryChannelRequest{
 		State:            true,
-		FilterConditions: q,
+		FilterConditions: q.Filter,
```

Comment:
what happens to pagination?

Reply:
the previous code didn't work.
we should probably fix that. maybe in another PR
filter_conditions must be map[string]interface{}

Reply:
nothing :).
this was broken.
we should fix/ add that in another PR

Reply:
OK, but how do you pass pagination now?

Reply:
we dont support paggination for queryChannels

Reply:
OK, existing code isn't working and error prone since enables passing wrong parameters but actually not supported. 

It would be nicer to get the bug fixed first, though.

Reply:
agree